.eslintrc

```diff
@@ -90,13 +90,7 @@
 			}
 		],
 		"linebreak-style": 0,
-		"lines-around-comment": [
-			2,
-			{
-				"beforeLineComment": true,
-				"allowBlockStart": true
-			}
-		],
+		"lines-around-comment": 0,
```

Comment:
I usually like this for readability, but I agree it can be annoying when commenting out a single line